portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -395,6 +399,18 @@ function createResult(result, wantedWords) {
   return [outputText, textElement];
 }
 
+function searchHash() {
+  const urlHash = window.location.hash;
+  if (urlHash) {
+    const hashArray = urlHash.split('#');
+    const outputText = decodeURI(hashArray[1]).substring(0, 5).toLowerCase();
```

Comment:
Why are we limiting the query string to 5 characters here?

Reply:
There were errors brought about making it longer because the spacing is different in the loaded pages vs the actual page due to css changes. I can try making the range larger.